core/models/identity.py

```diff
@@ -318,11 +318,11 @@ def json(self):
         }
 
     def __unicode__(self):
-        output = "%s %s - " % (self.provider, self.created_by.username)
-        output += "Credentials {"
-        for c in self.credential_set.order_by('key'):
-            output += "%s, " % (c.key,)
-        output = output[:-2] + "}"
+        output = "%s %s - Credentials" % (self.provider_id, self.created_by_id)
+        #output += "Credentials {"
+        #for c in self.credential_set.order_by('key'):
+        #    output += "%s, " % (c.key,)
+        #output = output[:-2] + "}"
```

Comment:
On second thought, this was something that was useful during early debugging, and now, is obsolete. I will remove it.